contrib/redhat/collectd.spec

```diff
@@ -2388,6 +2396,9 @@ fi
 %if %{with_write_log}
 %{_libdir}/%{name}/write_log.so
 %endif
+%if %{with_write_syslog}
```

Comment:
Please keep this list sorted

Reply:
Done

Reply:
no you didn't? write_syslog should come after write_sensu

Reply:
Sorry. Fixed